resources/views/service/centreusers/index.blade.php

```diff
@@ -59,6 +61,11 @@
                 @endforeach
             </tbody>
         </table>
+        <a href="{{ URL::route('admin.centreusers.download')}}" class="link" target="_blank">
```

Comment:
Check to see if they want it at the top or the bottom

Reply:
dropped a messaged on the Trello card. will wait for them before merging as I realised it's also a minor case of copy - what should the button say and how should we title the file to be downloaded.